packages/e2e-tests/specs/editor/blocks/navigation.test.js

```diff
@@ -211,6 +211,18 @@ async function getNavigationMenuRawContent() {
 // Disable reason - these tests are to be re-written.
 // eslint-disable-next-line jest/no-disabled-tests
 describe( 'Navigation', () => {
+	const contributorUsername = 'contributoruser';
```

Comment:
I wonder if we should make the username unique by adding some randomness, as one of the reasons WordPress would prevent creation of a user is a duplicate username, and that might happen if clean up of this test fails, or possibly another test using the same username.

Reply:
I added randomness using lodash. Does that work?

Reply:
This might be slightly over-engineered. I'd imagine a better solution would be having a proper error message if that happens. We are already running our tests in-band, so such things shouldn't happen. If they happen, it might be because our test cases aren't being correctly cleaned up, and we should fix them. But it's not that it's an anti-pattern or anything like that, I don't mind landing this 😅 .

Reply:
True, it shouldn't happen, but if it did I feel like it would still be really obscure and hard to track down.

I don't think `_.uniqueId` would do the job anyway, as that's just an incrementing integer. Another test in a new process would generate the same integer.